classes/class-onboard.php

```diff
@@ -72,84 +70,85 @@ public static function the_form() {
 				<?php
 				printf(
 					/* translators: %s: progressplanner.com link */
-					\esc_html__( 'We would love to send you awesome emails with your progress stats. Please enter your name and email below and submit the form. You\'ll create an account on %1$s:', 'progress-planner' ),
+					\esc_html__( ' We can send you weekly emails with your own to-do’s, your activity stats and nudges to keep you working on your site. To do this, we’ll create an account for you on %s.', 'progress-planner' ),
```

Comment:
Is the space at the beginning of this string on purpose? Or was it just a typo that we missed @jdevalk ? 🤔

Reply:
```suggestion
					\esc_html__( 'We can send you weekly emails with your own to-do’s, your activity stats and nudges to keep you working on your site. To do this, we’ll create an account for you on %s.', 'progress-planner' ),
```